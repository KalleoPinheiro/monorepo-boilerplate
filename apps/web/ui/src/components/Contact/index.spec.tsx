import '@testing-library/jest-dom/extend-expect'
import { render, screen } from '@testing-library/react'
import React from 'react'
import Contact from '.'

describe('Main', () => {
  it('should render successfully', () => {
    render(<Contact />)
    expect(
      screen.getByRole('heading', { name: /Contato/i }),
    ).toBeInTheDocument()
    expect(screen.getByRole('heading', { name: /Contato/i })).toMatchSnapshot()
  })
})
