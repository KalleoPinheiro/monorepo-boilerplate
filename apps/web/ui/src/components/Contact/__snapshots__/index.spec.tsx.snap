// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main should render successfully 1`] = `
<h1>
  Contato
</h1>
`;
